data_discovery_ai/utils/preprocessor.py

```diff
@@ -251,15 +264,15 @@ def keywords_formatter(text: Union[str, List[dict]], vocabs: List[str]) -> List[
         keywords = ast.literal_eval(text)
     k_list = []
     for keyword in keywords:
-        for concept in keyword["concepts"]:
-            if keyword["title"] in vocabs and concept["id"] != "":
-                con = Concept(
-                    id=concept["id"].lower(),
-                    url=concept["url"],
-                    vocab_type=keyword["title"],
-                )
-                concept_str = con.to_json()
-                k_list.append(concept_str)
+        if keyword.get("concepts") is not None:
+            for con in keyword.get("concepts"):
```

Comment:
use `concept` , not `con` for readability

Reply:
changed to `concept` and `conceptObj` accordingly